modules/installation-vsphere-regions-zones.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+//* installing-vsphere-installer-provisioned-customizations.adoc [IPI]
+//* installing-vsphere-installer-provisioned-network-customizations.adoc [IPI]
+//* installing-vsphere.adoc [UPI]
+//* installing-vsphere-network-customizations.adoc [UPI]
+//* installing-restricted-networks-installer-provisioned-vsphere.adoc [IPI]
+//* installing-restricted-networks-vsphere.adoc [IPI]
+
+:_content-type: CONCEPT
+[id="installation-vsphere-regions-zones_{context}"]
+= VMware vSphere region and zone enablement
+
+You can deploy an {product-title} cluster to multiple vSphere datacenters that run in a single VMware vCenter. Each datacenter can run multiple clusters. This configuration reduces the risk of a hardware failure or network outage causing your cluster to fail.
+
+You can also specify multiple regions and zones on a cluster after installation. You must upgrade your existing cluster to {product-title} {product-version} before you configure a multiple region and zone environment.
```

Comment:
I might need to remove based on comment on https://issues.redhat.com/browse/OSDOCS-5237